blackberggroup/VUHC
# feat-rl-6566432060-Uber-Health-Updates

## What type of PR is this? (check all applicable)


updates to index page based on monday ticket requests.=

## Related Tickets & Documents
- Monday Task #6566432060
- https://blackberg-group.monday.com/boards/1931719392/pulses/6566432060

## QA Instructions, Screenshots, Recordings

ensure page loads. verify page updates of title and subtitle/bullets as well as the multimedia additions match this (test links): https://blackberg-group.monday.com/boards/1931719392/pulses/6566432060?asset_id=1436525871

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _site/assets/documents/VHA-Uber-Health-Connect-Initiative-Final-Report.pdf
 - _site/assets/documents/VHA-Uber-Health-Connect-Initiative-Playbook.pdf
 - _site/assets/img/media-1.png
 - _site/assets/img/media-2.png
 - _site/assets/img/media-va-uber-health-connect-visn8-roadshow
 - copy.png
 - assets/documents/VHA-Uber-Health-Connect-Initiative-Final-Report.pdf
 - assets/documents/VHA-Uber-Health-Connect-Initiative-Playbook.pdf
 - assets/img/media-1.png
 - assets/img/media-2.png
 - _includes/home/announcements-events.html
 - _includes/home/multimedia.html
 - _site/index.html
 - assets/.DS_Store
